File: MyBatisSpringJavaInit/src/test/java/MainTest.java
import org.junit.jupiter.api.Test;
import org.springframework.context.ApplicationContext;
import org.springframework.context.support.ClassPathXmlApplicationContext;
import pers.init.dao.TestDao;

public class MainTest {

    @Test
    public void test()
    {
        ApplicationContext context = new ClassPathXmlApplicationContext("applicationContext.xml");
        TestDao dao = (TestDao) context.getBean("testDao");
        System.out.println(dao.selectByPrimaryKey(1));
    }

}

File: MyBatisSpringKotlinInit/src/test/kotlin/MainTest.kt
import org.junit.Test
import org.springframework.context.ApplicationContext
import org.springframework.context.support.ClassPathXmlApplicationContext
import pers.init.dao.TestDao

class MainTest {
    @Test
    fun test()
    {
        val context: ApplicationContext = ClassPathXmlApplicationContext("applicationContext.xml")
        val dao = context.getBean("testDao") as TestDao
        println(dao.selectByPrimaryKey(1))
    }
}